src/senaite/jsonapi/api.py

```diff
@@ -1451,6 +1454,70 @@ def get_registry_records_by_keyword(keyword=None):
     return found_registers
 
 
+def get_settings_by_keyword(keyword=None):
+    """Get the settings associated to the specified keyword or, if
+     keyword is None, get all the settings.
+
+    :param keyword: settings to be retrieved
+    :return: dictionary with the settings plus a key to identify from which
+    keyword where retrieved.
+    """
+    key_to_ischema = {
+                      'mail':        [cp.mail.IMailSchema],
+                      'calendar':    [cp.calendar.ICalendarSchema],
+                      'ram':         [cp.ram.IRAMCacheSchema],
+                      'language':    [cp.language.ILanguageSelectionSchema],
+                      'editing':     [cp.editing.IEditingSchema],
+                      'usergroups':  [cp.usergroups.IUserGroupsSettingsSchema,
+                                      cp.usergroups.ISecuritySchema,],
+                      'search':      [cp.search.ISearchSchema],
+                      'filter':      [cp.filter.IFilterAttributesSchema,
+                                      cp.filter.IFilterEditorSchema,
+                                      cp.filter.IFilterSchema,
+                                      cp.filter.IFilterTagsSchema],
+                      'maintenance': [cp.maintenance.IMaintenanceSchema],
+                      'markup':      [cp.markup.IMarkupSchema,
+                                      cp.markup.ITextMarkupSchema,
+                                      cp.markup.IWikiMarkupSchema,],
+                      'navigation':  [cp.navigation.INavigationSchema],
+                      'security':    [cp.security.ISecuritySchema],
+                      'site':        [cp.site.ISiteSchema],
+                      'skins':       [cp.skins.ISkinsSchema],
+                      }
```

Comment:
Just PEP8 cosmetics;)

Indent the inner block 4 spaces from `key_to_ischema`, e.g.
```python
key_to_ischema = {
    'mail': [cp.mail.IMailSchema],
    'calendar': [cp.calendar.ICalendarSchema]
}
```
and do not align the values, but put them one space after the key.